src/server/pfs/reduce.go

```diff
@@ -16,6 +16,7 @@ func ReduceRepoInfos(repoInfos []*pfs.RepoInfo) []*pfs.RepoInfo {
 			continue
 		}
 		reducedRepoInfo.SizeBytes += repoInfo.SizeBytes
+		reducedRepoInfo.Provenance = repoInfo.Provenance
```

Comment:
Is it an invariant that the `Provenance` returned by each server should be the same?  If so, can we assert that?